PWGEM/PhotonMeson/Core/CutsLibrary.cxx

```diff
@@ -57,6 +57,14 @@ EMEventCut* o2::aod::pwgem::photon::eventcuts::GetCut(const char* cutName)
     return cut;
   }
 
+  if (!nameStr.compare("minbias_noTBForROFCut")) {
```

Comment:
Let's put like this:
"minbias" as sel8 and |Zvtx| < 10 cm only.
"minbias_notfb" as sel8 and |Zvtx| < 10 cm and No TFB.
"minbias_notfb_noitsrofb" as sel8 and |Zvtx| < 10 cm and No TFB and No ITS ROFB.

Reply:
Yes, I agree. I implemented it in the updated version